app-web/src/components/Home/ResourcePreview.js

```diff
@@ -89,42 +89,40 @@ export const ResourcePreview = ({ title, link, resources, filters, amountToShow,
   let [showCount, setCount] = useState(amountToShow);
   let [seeMoreResults, setSeeMore] = useState(seeMore);
   let [resourcesToShow, setResources] = useState(resources);
-  let [activeFilter, setFilter] = useState('All');
+  let [activeFilter, setFilter] = useState(['All']);
+  // let activeFilter = ['All'];//give up on using hook for this, it doesn't work!
 
   //sets the amount of resources to show, allowing users to 'see more' if its appropriate
   const updateCount = () => {
     //show 6 more results
     setCount(showCount + 6);
```

Comment:
the number '6' is more or less a magic number, we should abstract this into a configuration/constant variable that is passed in as a argument.